bin/join-metadata-and-clades

```diff
@@ -16,6 +16,10 @@ reference_day = datetime(2020,1,1).toordinal()
 
 column_map = {
     "clade_legacy": "Nextstrain_clade",
+    "clade_nextstrain": "year_letter_clade",
+    "clade_who": "clade_who",
+    "clade_legacy": "clade_legacy",
+    "clade_legacy": "display_clade",
```

Comment:
> Finally it gives us a new column display_clade that we can modify to help us easily show whatever type of clade labels we want in the future, without breaking workflows 

This colum will be identical to "Nextstrain_clade" (both are just using Nextclade's "clade_legacy"). Is the idea to introduce this column in this PR in preperation for a future new column in Nextclade which will become the "display_clade"? Or will there be code in ncov-ingest to generate this name?

As far as I understand it a value such as “21K (BA.1)” doesn't exist in the metadata ncov-ingest produces as it currently stands.

Reply:
Yea, it indeed may be best to not have the `display_clade` column at all, and have this added internally in `ncov` (see Tom's arguments above). 

I mostly added this because from the complexity of changing stuff in ncov it seemed like a possibly easy solution, but if we can come up with better ways to do this, I'd be happy to drop having this column in metadata!